lib/main.js

```diff
@@ -318,8 +362,8 @@ function signText(text) {
       logPRError();
       return ERROR_INTERNAL;
     }
-    log("using '" + certName + "'");
-    cert = CERT_FindUserCertByUsage(certDB, certName, certUsageEmailSigner,
+    log("using '" + certName.nickname + "'");
+    cert = CERT_FindUserCertByUsage(certDB, certName.nickname, certUsageEmailSigner,
```

Comment:
I've been trying to keep lines < 80 characters - this looks a bit long (it's not a big deal, but it would be nice if we could keep to that style).